### PR TITLE
Update missing module command mappings

### LINKED ISSUE #4814: Update missing module command mappings
### Expected behavior

- Add all known `module` `command` mappings
- Handle `unknown` `module` `command` mappings
- Application shouldn't crash
- We could use `/api/v3/network/status` `moduleCommands` for handling the mapping dynamically

### Actual behavior

- Wallet is unable to render dashboard or transactions table when an `unknown module command` response needs to be processed
Screenshot 2023-02-14 at 7 14 08 PM

### Steps to reproduce

- Submit any transactions whos mapping doesn't exists e.g `Legacy RegisterKeys`
- Observe the dashboard crashes

### Which version(s) does this affect? (Environment, OS, etc...)
3.0

### PR DESCRIPTION
**Pull request by oskarleonard**
### What was the problem?

This PR resolves #4814

### How was it solved?
- Adding missing module commands

### How was it tested?
Manually

### CODE REVIEW

File: src/modules/common/__fixtures__/mockCommandParametersSchemas.js [hunk 1]
```diff
@@ -2389,54 +2389,6 @@ export const mockCommandParametersSchemas = {
           },
         },
       },
-      {
```
Comment (ManuGowda) [inline src/modules/common/__fixtures__/mockCommandParametersSchemas.js:2392]: why is this removed?

Comment (oskarleonard) [reply src/modules/common/__fixtures__/mockCommandParametersSchemas.js:2392]: isnt that renamed to `transferCrossChain` ?

**Review by ManuGowda — APPROVED** [review]
